src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$size--block: 100px;
+$start:
+  'mon' 0,
+  'tue' 101px,
+  'wed' 202px,
+  'thu' 303px,
+  'fri' 404px,
+  'sat' 505px,
+  'sun' 606px;
```

Comment:
make numbers from 0 to 7 and count on loop. it will be more flexible.
if will change gap or cell width - will be enough to change in one place